custom_components/alexa_media/media_player.py

```diff
@@ -1375,7 +1375,9 @@ async def async_send_dropin_notification(self, message, **kwargs):
             )
 
     @_catch_login_errors
-    async def async_play_tts_cloud_say(self, public_url, media_id, **kwargs):
+    async def async_play_tts_cloud_say(
+        self, media_type, public_url, media_id, **kwargs
```

Comment:
Do we need to bump the min ha version in manifest.json to account for media_type? I don't think it used to exist when the original code was written.

Reply:
I am not sure about this one.

Reply:
The current min version mentioned in hacs.json is 2024.2.0b0 which is relatively new so it should work I think.

Reply:
media_type was already being used in this custom integration so this shouldn't be a problem